Package.swift

```diff
@@ -1,4 +1,17 @@
 // swift-tools-version:5.1
+
+/*
+ * Copyright (c) 2020-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
 // The swift-tools-version declares the minimum version of Swift required to build this package.
```

Comment:
You might want to move this line up to line 2, to make the standard swift tools header comments sit together.

Reply:
Good input. Done